index.js

```diff
@@ -1,33 +1,26 @@
-// Without "./", Node looks
-// in its global module folder
-const http = require("http");
-function handleRequestAndResponse(req, res) {
-    console.log(req.url);
-    const timestamp = new Date();
-    console.log("Received a Request");
-    res.end(`<h1>Hello World ${timestamp}</h1>`);  // .end() sends the response.
-                                          // It "tapes up the box and ships it".
+const nerds = require(`nerds`);
+const http = require(`http`);
+
+function getRandomInt(max) {
```

Comment:
Not necessary for this exercise, but functions like this are a good candidate for placing in a separate module. (Since it could be used in multiple places in a larger project.)